timesketch/lib/utils.py

```diff
@@ -181,3 +187,55 @@ def get_validated_indices(indices, sketch_indices):
     if exclude:
         indices = [index for index in indices if index not in exclude]
     return indices
+
+
+def send_email(subject, body, to_username, html=False):
+    """Send email using configure SMTP server.
+
+    Args:
+        subject: Email subject string.
+        body: Email message body.
+        to_username: User to send email to.
+        html: Boolean indicating if the email body should be sent as html.
+
+    Raises:
+        RuntimeError if not properly configured or if the recipient user is no
+        in the whitelist.
+    """
+    email_enabled = current_app.config.get('ENABLE_EMAIL_NOTIFICATIONS')
+    email_domain = current_app.config.get('EMAIL_DOMAIN')
+    email_smtp_server = current_app.config.get('EMAIL_SMTP_SERVER')
+    email_from_user = current_app.config.get('EMAIL_FROM_ADDRESS')
+    email_user_whitelist = current_app.config.get('EMAIL_USER_WHITELIST', [])
+
+    if not email_enabled:
+        raise RuntimeError('Email notifications are not enabled, aborting.')
+
+    if not email_domain:
+        raise RuntimeError('Email domain is not configured, aborting.')
+
+    if not email_smtp_server:
+        raise RuntimeError('Email SMTP server is not configured, aborting.')
+
+    # Only send mail to whitelisted usernames.
+    if to_username not in email_user_whitelist:
+        raise RuntimeError('User {0:s} is not in the email whitelist, '
```

Comment:
why raise a RuntimeError if the user is not whitelisted?

Does this crash things on the other end, or?

Reply:
I agree, removed the raising.